hmmer/3.3/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# hmmer
+# http://hmmer.org/
+# v3.3
+
+# base image: Ubuntu
+FROM ubuntu:xenial
+
+# File Author / Maintainer
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="hmmer"
+LABEL software.version="3.3"
+LABEL description="This software provides HMMer for hmmsearch etc.!"
+LABEL website="https://github.com/StaPH-B/docker-builds"
+LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"
```

Comment:
`LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"` -->`LABEL license="http://eddylab.org/software/hmmer/Userguide.pdf"`

We usually provide a link to the license file on GitHub repo's, but again hmmer is not on your typical GitHub repo. So in this case, I recommend linking to the User guide, where it does mention the license for Hmmer - "3-Clause BSD open source license"